Shipfinity.Domain/Models/Address.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Shipfinity.Domain.Enums;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shipfinity.Domain.Models
+{
+    public class Address : BaseEntity
+    {
+        public string AddressLine { get; set; }
```

Comment:
Limit the characters of these string values so they are not created as varchar(MAX) example Zipcode should be fairy short MaxLength(10) maybe or 15, AddressLine should be around MaxLegnth(100) so on

Reply:
done